docs/dictionary/command/ask-file.lcdoc

```diff
@@ -42,13 +42,16 @@ suggested file name is provided in the defaultFilePath, it appears in
 the file name box.
 
 types:
-The  types parameter specifies which extensions are available at the
+The types parameter specifies which extensions are available at the
 bottom of the dialog box. A file type consists of an optional
 description, a line feed or comma, and a file extension specification:
-[description],*.extension You can include more than one file type by
-separating the file types with commas.>*Cross-platform note:* The types
-parameter applies only to Windows systems. On Mac OS and Unix systems,
-this parameter has no effect.
+  [description],*.extension 
```

Comment:
I think you need to add an extra line above and below this one for the code block to work